llvm/include/llvm/TableGen/Record.h

```diff
@@ -1650,6 +1650,8 @@ class Record {
     DumpInfo(SMLoc Loc, Init *Message) : Loc(Loc), Message(Message) {}
   };
 
+  enum RecordKind { RK_Normal, RK_Anonymous, RK_Class, RK_MultiClass };
```

Comment:
May I suggest the following rename? Seems to be more straightforward...
```suggestion
  enum RecordKind { RK_Def, RK_AnonymousDef, RK_Class, RK_MultiClass };
```